app/javascript/controllers/delete_evaluator_modal_controller.js

```diff
@@ -43,30 +43,29 @@ export default class extends Controller {
   deleteEvaluator(forceDelete = false) {
     const csrfToken = document.querySelector('meta[name="csrf-token"]').content
   
-    fetch(`/phases/${this.phaseIdValue}/manage_evaluators/${this.evaluatorIdValue}`, {
+    fetch(`/phases/${this.phaseIdValue}/evaluators/${this.evaluatorIdValue}`, {
       method: 'DELETE',
       headers: { 'Content-Type': 'application/json', 'X-CSRF-Token': csrfToken },
       body: JSON.stringify({ evaluator_type: this.evaluatorTypeValue, phase_id: this.phaseIdValue, force_delete: forceDelete })
     })
     .then(response => {
-      if (!response.ok) {
-        throw new Error('Network response was not ok')
-      }
-      const data = response.json()
+      if (!response.ok) { throw new Error('Network response was not ok') }
+      return response.json()
+    })
+    .then(data => {
```

Comment:
I broke this on your PR, the DELETE request is working but the response wasn't being handled correctly with promises so I added this back, sorry!